python/tk_multi_reviewsubmission/submitter.py

```diff
@@ -49,8 +49,8 @@ def submit_version(self, path_to_frames, path_to_movie, thumbnail_path, sg_publi
             "sg_status_list": self.__app.get_setting("new_version_status"),
             "entity": ctx.entity,
             "sg_task": sg_task,
-            "sg_first_frame": first_frame,
-            "sg_last_frame": last_frame,
+            "sg_first_frame": head_in,
```

Comment:
This should not be necessary. If the first frame should be head_in, then we should be shooting a movie from head_in, in which case first_frame and head_in are the same. The same is true for tail_out. Please revert.

Reply:
first_frame here corresponds to cut_in. I'm using head_in to tail_out because on lemony, we add 16 black/slate frames at either end, and RV (launched via Shotgun) uses the first/last frame fields of a version entity to display the frame number correctly.
If we put in cut_in/cut_out, RV doesn't just play those n frames, but plays the first n frames (including the 16 slate frames), which we don't want to do.
If we leave these blank (that's how it is in the LA shotgun), RV picks up the frame range computed from the timecode and that makes it hard to navigate (1000 becomes 86400, so if they want to view some frame, say 1019, they can't click on 1019 on the timeline, but would have to calculate how far it is from 86400)

Reply:
Of course, all this may change if we launch RV in a different way. So I guess we should look at the behaviour after that update before deciding what is best...?

Reply:
> first_frame here corresponds to cut_in

This is the error. first frame should always correspond to head_in, not cut_in. If this is not the case, we are passing the wrong value to the api entry point from wherever...

Reply:
Updated app.py to pass head_in if available, instead of cut_in.